odxtools/dynamiclengthfield.py

```diff
@@ -49,11 +49,71 @@ def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
 
     def convert_physical_to_bytes(
         self,
-        physical_value: ParameterValue,
+        physical_values: ParameterValue,
```

Comment:
Sure about the argument name being plural?

Reply:
I suppose that this a matter of taste: since this is a list, the plural is justified. Anyway, I changed it to singular. (the plural comes from the fact, that the method here was initially pasted from EndOfPduField...)